concepticondata/references/references.bib

```diff
@@ -1692,4 +1692,13 @@ @article{Bromley1967
   Title                               = {The linguistic relationships of Grand Valley Dani: A lexico-statistical classification},
   Volume                              = {37},
   Year                                = {1967}
-}
\ No newline at end of file
+}
+
+@article{VanderWouw1974,
```

Comment:
It's capitalization, why the checks fail. I guess, I'd use something like VanDerWouw, but there is no strict rule for that in the current practice, and vanderWouw/VanderWouw work just as fine.